mobilestyle.css

```diff
@@ -52,12 +45,13 @@ a {
 }
 
 .menuNav {
-  margin-right: 4%;
+  flex: 1;
   margin-top: 2%;
+  margin-left: 20%;
 }
 
 .menuImg {
-  width: 30px;
+  width: 25px;
 }
```

Comment:
it's advisable to make use of percentage in place of pixels for a more responsive page